hapi-base/src/main/java/ca/uhn/hl7v2/parser/FixFieldDataType.java

```diff
@@ -50,6 +50,13 @@ private FixFieldDataType() {}
      */
     public static final String DEFAULT_OBX2_TYPE_PROP = "ca.uhn.hl7v2.model.varies.default_obx2_type";
 
+    /**
+     * System property key: If this is not set to true, it will be a false.
+     *
+     * OBX-2 enable  OBX-2 default value.
+     */
+    public static final boolean ENABLE_DEFAULT_OBX2_DEFAULT_VALUE = "ca.uhn.hl7v2.model.varies.enable.obx2.default.value";

```

Comment:
The major purpose of the fix() method is to take a VARIES field and fix it to a particular data type. Setting the field representing the data type to a default value is sort of just a side effect.  I think it would be cleaner if we kept on fixing the OBX-5  (presumably to TX in our current case) and just did not auto-populate the OBX-2.  However if we want to stop fixing the OBX-5 (what is the impact of this?) could we give the name of our new system property something that reflects this?

Reply:
Updated. Thanks.